website/static/js/pages/project-base-page.js

```diff
@@ -32,4 +32,43 @@ $(document).ready(function() {
     $.getJSON(node.urls.api, function(data) {    
         $('body').trigger('nodeLoad', data);
     });
+
+    var self = this;
+    var THRESHOLD_SCROLL_POSITION  = 50;
+    var SMALL_SCREEN_SIZE = 767;
+    var NON_NAV_TOP_MARGIN = 50;
+    var NAV_MAX_TOP_MARGIN = 95;
+    self.adjustPanelPosition = function() {
+        var bodyWidth = $(document.body).width();
+        var scrollTopPosition = $(window).scrollTop();
+        if (bodyWidth <= SMALL_SCREEN_SIZE) {
+            if (scrollTopPosition >= THRESHOLD_SCROLL_POSITION) {
+                $('.cp-handle').css('margin-top', NON_NAV_TOP_MARGIN);
+            }
+            else {
+                $('.cp-handle').css('margin-top', NAV_MAX_TOP_MARGIN - scrollTopPosition);
+            }
+        }
+    };
+    var THROTTLE = 10;
+
+    self.adjustPanelPosition(); /* Init when refreshing the page*/
+    $(window).scroll(
+        $osf.debounce(
+            function() {
+                self.adjustPanelPosition();
+    }, THROTTLE));
+
+    THROTTLE = 50;
```

Comment:
For ease of understanding, it's better to have separate variables rather than reassigning, e.g. `ADJUST_PANEL_DEBOUNCE` and `RESIZE_DEBOUNCE`.